packages/cli/src/ceramic-daemon.ts

```diff
@@ -284,6 +285,17 @@ export class CeramicDaemon {
     )
     const ipfsId = await ipfs.id()
 
+    // we need to change the values before processing the config, so we keep track to log the warning after we have logging set up
+    const desiredIpfsClient = EnvironmentUtils.useRustCeramic() ? 'ceramic-one' : 'kubo'
```

Comment:
does this need to happen in the ceramic-daemon or can it be pushed down into ceramic.ts?

Reply:
it needs to happen here because we have to change it before we process the config otherwise we get misconfigured pubsub/recon things and we don't tell ceramic whether we expected remote or not currently, which could probably change as it would improve the messages later, but I tried not to change more than I needed

Reply:
we could probably update the `CeramicConfig` object to include the information if we want to go that route

Reply:
this is fine

Reply:
as discussed, we actually don't care about remote at this point because it's always remote, but `_processConfig` requires the variable to be set correctly for pubsub and recon things to work, so it would have to happen in there (or use `create` from the daemon) which probably isn't worth it right now